docs/configuration-gps.md

```diff
@@ -10,11 +10,12 @@ Two GPS protocols are supported. NMEA text and UBLOX binary.
 
 Enable the GPS as follows:
 
-1. Connect your GPS to the serial port for GPS.
-1. Enable the `feature GPS`
-1. Set your GPS baud rate
-1. set the `GPS provider`
-1. save and reboot.
+1. Connect your GPS to the desired serial port for GPS.
+2. [Configure serial port to be used by GPS.](configuration-serial.md)
```

Comment:
But where is that file  ?

Reply:
It is located [here](https://github.com/raul-ortega/amv-open360tracker-32bits/wiki/Telemetr%C3%ADa-de-entrada:-Configuraci%C3%B3n-del-puerto-serie-0-%28UART1%29) ,and It is pending for translation to english from my side. I'll do it today to finish with thi topic. I merged the other changes of you PR. Thanks so much.

Reply:
I apollogyze, is [this link]https://github.com/raul-ortega/amv-open360tracker-32bits/wiki/GPS-local:-configuraci%C3%B3n)

Reply:
Wow. There's a lot of info that's worth translating. I I only knew spanish... ;-)